best-practices/README.md

```diff
@@ -13,6 +13,12 @@ General
 * [Keep the code simple](http://www.readability.com/~/ko2aqda2).
 * Avoid premature optimizations. Get a version 1.0 out there as soon as you
   can. Until you have some users to measure, you're optimizing based on guesses.
+* Use `bin/setup` for initial setup of the project and for development
+  environment seed data.
+* Use `bin/server` for starting all the processes required to run the project
+  in a development environment, when suitable.
+* Use `bin/deploy` for deploying the project to staging and production
```

Comment:
You can do the equivalent of what parity does on CI yes (that was the question right?)

That wasn't my point. I only deploy via parity because my projects don't have auto-deploys to production, and I find it easier to deploy via CLI rather than going to the CI. But that setup should be done regardless